src/librustc/lint/builtin.rs

```diff
@@ -125,6 +125,12 @@ declare_lint! {
     "detect private items in public interfaces not caught by the old implementation"
 }
 
+declare_lint! {
+    pub EXTERNAL_PRIVATE_DEPENDENCY,
```

Comment:
Lint naming conventions - https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#lints.
`EXTERNAL_PRIVATE_DEPENDENCY` -> `EXTERNAL_PRIVATE_DEPENDENCIES`.

Also, what the lint wants to say is probably "exported private dependencies" rather than "external private dependencies".

Reply:
I had copied this [from the RFC](https://github.com/rust-lang/rfcs/blob/master/text/1977-public-private-dependencies.md#how-we-teach-this), but I can change it.